tamr_unify_client/models/dataset/usage.py

```diff
@@ -0,0 +1,39 @@
+from tamr_unify_client.models.dataset.use import DatasetUse
+
+
+class DatasetUsage:
+    """
+    The usage of a dataset and its downstream dependencies.
+
+    See https://docs.tamr.com/reference#retrieve-downstream-dataset-usage
+
+    :param client: Delegate underlying API calls to this client.
+    :type client: :class:`~tamr_unify_client.Client`
+    :param data: The JSON body containing usage information.
+    :type data: :py:class:`dict`
+    :param alias: API alias for this usage.
+    :type alias: str
+    """
+
+    def __init__(self, client, data, alias):
```

Comment:
Warrants a comment about why this is not a BaseResource

Reply:
It should have been I guess.